components/Banner/src/MDCBannerView.m

```diff
@@ -195,15 +195,11 @@ - (void)setUpImageViewConstraints {
   self.imageViewConstraintLeading =
       [self.imageView.leadingAnchor constraintEqualToAnchor:self.layoutMarginsGuide.leadingAnchor
                                                    constant:kLeadingPadding];
-  self.imageViewConstraintTopSmall =
-      [self.imageView.topAnchor constraintEqualToAnchor:self.layoutMarginsGuide.topAnchor
-                                               constant:kTopPaddingSmall];
   self.imageViewConstraintTopLarge =
       [self.imageView.topAnchor constraintEqualToAnchor:self.layoutMarginsGuide.topAnchor
                                                constant:kTopPaddingLarge];
-  self.imageViewConstraintBottom =
-      [self.imageView.bottomAnchor constraintEqualToAnchor:self.layoutMarginsGuide.bottomAnchor
-                                                  constant:-kBottomPadding];
+  self.imageViewConstraintCenterY =
+      [self.imageView.centerYAnchor constraintEqualToAnchor:self.buttonContainerView.centerYAnchor];
```

Comment:
Should this be centered to self.centerYAnchor?

Reply:
Button, text and button are not centered to the view on Y axis according to the guideline. There is a one point off.